src/lib.rs

```diff
@@ -41,5 +41,11 @@ pub const CONFIG_ENV: &str = "CONFIG";
 /// Environment variable for setting the authentication token.
 pub const AUTH_TOKEN_ENV: &str = "AUTH_TOKEN";
 
+/// Environment variable for the path to a file containing multiple authentication token.
+pub const AUTH_FILE_ENV: &str = "AUTH_FILE";
+
 /// Environment variable for setting the deletion token.
 pub const DELETE_TOKEN_ENV: &str = "DELETE_TOKEN";
+
+/// Environment variable for the path to a file containing multiple deletion token.
+pub const DELETE_FILE_ENV: &str = "DELETE_FILE";
```

Comment:
I was about to merge this but then had an epiphany, `DELETE_FILE` sounds more like "delete this file" rather than "a file that contains the deletion tokens". I think we should grammatically fix this while we have the chance :) So I propose:

- `AUTH_FILE` -> `AUTH_TOKENS_FILE`
- `DELETE_FILE` -> `DELETE_TOKENS_FILE`

Thoughts? @Nydragon @tessus

Reply:
Completely agree, actually chose that name at the beginning, but replaced it with the current one because it was shorter.

Will update that as soon as I'm back home!

Reply:
Shorter names only make sense, if it has to be used all the time....

Reply:
Changed the names